teps/0115-tekton-catalog-git-based-versioning.md

```diff
@@ -0,0 +1,398 @@
+---
+status: implementable
+title: Tekton Catalog Git-Based Versioning
+creation-date: '2022-07-12'
+last-updated: '2022-08-04'
+authors:
+- "@jerop"
+- "@vdemeester"
+- "@vinamra28"
+- "@QuanZhang-William"
+- "@wlynch"
```

Comment:
@wlynch fyi added you as a co-author - thank you for the many contributions here!